docs/lang/articles/misc/profiler.md

```diff
@@ -4,78 +4,122 @@ sidebar_position: 4
 
 # Profiler
 
-Taichi's profiler can help you analyze the run-time cost of your
-program. There are two profiling systems in Taichi: `KernelProfiler` and
-`ScopedProfiler`.
+## Overview
+High-performance numerical computation is one of the design key application domains of Taichi.
```

Comment:
```suggestion

High-performance numerical computation is one of the key application domains of Taichi.
```

(not sure if I understood the sentence correctly...)

Reply:
Probably because I merged the previous suggestion `key application domains` but haven't deleted `design`.